### DISCUSSION TITLE
To stupid to add my Street?

### DISCUSSION THREAD

**Discussion by Stephan-4711** [mampfes/hacs_waste_collection_schedule#2308] [General]
My City is inside of the configuration Germany Celle
Then I get the company ZAC wich is correct.
When I write my city and street I always get the error Street not found.
I tried Captial Letter, non captial letter str. and straße but nothing works.
Is there a way to look up all Streets?
Or what am I doing wrong?

**Comment by Stephan-4711** [comment]
Does it have to do with the fact that mymuell offers an own app Celle?

**Comment by Stephan-4711** [comment]
Ok was able to solve my self.
The App by MyMuell has my street name and behind the street there is my district added.
Maybe you can add something like a search function?

**Reply by 5ila5** [reply]
It's not that easy but something like this is on our to-do list

**Reply by Stephan-4711** [reply]
Amazin, may I ask another question. You see the dates in the screenshot. I first tried how many days until the emptying date. But they showed 3 days left 4 days left and 3 days again, but as you see this really doesn't fit to the correct dates
![Screenshot_20240719-184012](https://github.com/user-attachments/assets/824ad7d4-68b9-4963-aaa0-fc854a7c71f5)

**Reply by 5ila5** [reply]
Not quite shure what you did, but this works for me
![image](https://github.com/user-attachments/assets/a73eaafb-78f6-48d5-a274-aa395e67a577)
![image](https://github.com/user-attachments/assets/3bbff3d6-11bb-476e-8ef3-92508ff62ee6)
![image](https://github.com/user-attachments/assets/54888c7b-12d5-43bd-8a96-0f8d399b68de)

**Comment by Stephan-4711** [comment]
Here it looks the same, the date it self is correct.
But the in X days is wrong

**Reply by 5ila5** [reply]
I dont understand the in .. days are correct (Saturday, Sunday is cut off in the image), Gelber sack is in 11 days (time of picture), papier is in 3 days and Restmüll is in 14 days (note after the time set in day_switch time (defaults to 10:00 AM), the sensor will ignore events on the current day and swtich to the next one)
